mitodl/open-discussions
# Ab/use new course search utils

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/3595

#### What's this PR do?
This PR updates ocw-hugo-themes to use the new shared query generation code

#### How should this be manually tested?
Run open-discussions locally. Verify that the both /search and /learn/search work as expected
